### PR TITLE
Finalizing Template Folder

### PR DESCRIPTION
**Pull request by LinnierGames**
#### Current State <span><a href="#details">( scroll down )</a></span>
- [X] Final
#### Todos for the reviewer
- [x] Look at Changes Types
#### Deploy Notes

To be merged with `origin/master`

<h1 id="details">Details about this Pull Requset</h1>

#### What's this Pull Request do?

Add more stuff from #4 
#### Changes or additions made:
- [improved] Changes types ( added, improved, fix )
- [fix] Removed FORMATS.md since there was no need of it

### CODE REVIEW

**Comment by blhough** [comment]
nice reference to past PR

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
+
+#### Requirments before I make this Pull Request
+
 - [X] I am submitting a pull request :)
 - [ ] There is documentation in all methods and classes I've created
 - [ ] Merging conflicts have been resolved
-    - [ ] Pulled new code into `Development`
-    - [ ] Merged `Development` into `Feature Branch`
-    - [ ] Fixed `Development` into `Feature Branch`
+    - [ ] Pulled new code into `local/dev` from `origin/dev`
```
Comment (blhough) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: nicely said!

Comment (LinnierGames) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: lol :D hopefully we all know `local` and `origin`

**Comment by LinnierGames** [comment]
@whothat should i keep deleting the branch from `origin` and republish every time wanna add changes or the repo?

**Comment by blhough** [comment]
@LinnierGames I think it would be best to delete the branch on github after it has been merged to keep it clean.

**Comment by LinnierGames** [comment]
sounds good! gotta live up to the [workflow](https://github.com/SRJC-Computer-Science-Club/micromouse/blob/master/workflow.png)

**Pull request merged**